ocaml/jane/doc/proposals/data-race-freedom.md

```diff
@@ -94,11 +94,11 @@ annotation.
 As a preview, we are going to describing 5 mode axes in this document,
 which have the following modes ordered by increasing strictness.
 
-| Locality | Sync   | Uniqueness | Linearity  | Readonly  |
-| -------- | ------ | ---------- | ---------- | --------- |
-| Local    | Unsync | Shared     | Once       | Readonly  |
-| Global   | Sync   | Exclusive  | Separate   | Readwrite |
-|          |        | Unique     | Many       |           |
+| Locality | Portable   | Uniqueness | Linearity  | Readonly  |
+| -------- | ---------- | ---------- | ---------- | --------- |
+| Local    | Unportable | Shared     | Once       | Readonly  |
```

Comment:
This should say `Nonportable` not `Unportable`

Reply:
This was fixed in #2634